lib/Services/auth.dart

```diff
@@ -1,17 +1,17 @@
-import 'package:app_mental/model/user.dart';
+import 'package:app_mental/helper/helperfuncions.dart';
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter_chat_types/flutter_chat_types.dart' as types;
 import 'package:flutter_firebase_chat_core/flutter_firebase_chat_core.dart';
 
 class AuthMethods {
-  final FirebaseAuth _auth = FirebaseAuth.instance;
+  //final FirebaseAuth _auth = FirebaseAuth.instance;
 
   // UserApp? _userFromFirebaseUser(User? user) {
   //   return user != null ? UserApp(userId: user.uid) : null;
   // }
 
-  Future<UserCredential?> signInWithEmailAndPassword(
+  /*Future<UserCredential?> signInWithEmailAndPassword(
```

Comment:
Se não for usar mais, pode deletar

Reply:
Eu apaguei tudo, qualquer coisa me fala que eu restauro o que não deve ser apagado